src/utils/cleanIdentifierInputs.ts

```diff
@@ -1,10 +1,14 @@
 import { IdentifierInput } from "../types";
 
 export const cleanIdentifierInputs = (
-  input: IdentifierInput[]
+  input: IdentifierInput[],
+  optionalIdentifier: string[]
 ): IdentifierInput[] => {
   for (const identifierInput of input) {
     identifierInput.value = identifierInput.value.trim();
+    if (optionalIdentifier.includes(identifierInput.label)) {
```

Comment:
I do not think that this should be here, since it does not fall under cleaning identifier inputs. Shall we write another function to do this instead?

Reply:
if we do so, we will have multiple iterations of the same `identifierInput` tho. What do you think of refactoring the name `cleanIdentifierInputs` instead?

Reply:
Is it possible for identifier input labels to be the same? If so, what happens when there are identifier inputs with the same label but one is optional, but the other is not?